### ISSUE TITLE
[JAWS] Search results read "results found" instead of something useful

### ISSUE THREAD

**Issue by carlos-zamora** [microsoft/terminal#14153]
### Windows Terminal version

1.16.2642

### Other Software

JAWS (though this probably repros with any screen reader)

### Steps to reproduce

1. generate text (i.e. `ls`)
2. open search dialog (<kbd>ctrl+shift+f</kbd>)
3. search for something (i.e. `text`) NOTE: make sure there are results!

### Expected Behavior

the line of text should be read (i.e. `this is example text` where "text" is selected)

This gives context to the user

### Actual Behavior

reads "results available"

Most likely caused by https://github.com/microsoft/terminal/pull/12301

## Other feedback
- "no results found" is good!
- "findMatch" should work the exact same way
- when we loop around the buffer, we should say "back to top/bottom"

**Comment by carlos-zamora** [comment]
Throwing this in _next_ milestone. I don't think we'll have time to fix this for this coming release, but this is a decent dissatisfaction for screen reader users (I mean, find is basically useless tbh).

Feel free to change if you disagree.

**Comment by zadjii-msft** [comment]
I'm tossing this back to the backlog. Though, now we could actually announce how many results there were, so that's cool
